lambda/src/main/scala/pricemigrationengine/services/EnvConfig.scala

```diff
@@ -21,9 +23,9 @@ object EnvConfig {
   object zuora {
     val layer: Layer[ConfigFailure, ZuoraConfig] = ZLayer.fromZIO(
       for {
-        zuoraApiHost <- env("zuoraApiHost")
-        zuoraClientId <- env("zuoraClientId")
-        zuoraClientSecret <- env("zuoraClientSecret")
+        zuoraApiHost <- ZIO.succeed(SecretManagerClient.secrets.zuoraApiHost)
```

Comment:
This shouldn't be a `ZIO.succeed` unless there's absolutely no possibility of it failing